### PR TITLE
CB-1577-H2-driver-update

### PR DESCRIPTION
**Pull request by EmYar**

### CODE REVIEW

**Comment by ShadelessFox** [comment]
AFAIK you're not supposed to bump versions of plugins that are shipped with DBeaver.

It makes sense for plugins installed separately so that Eclipse can figure out that they were updated.

File: plugins/org.jkiss.dbeaver.ext.h2/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %Bundle-Name
 Bundle-SymbolicName: org.jkiss.dbeaver.ext.h2;singleton:=true
-Bundle-Version: 1.0.156.qualifier
+Bundle-Version: 1.0.157.qualifier
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.ext.h2/META-INF/MANIFEST.MF]: Please don't change bundle version manually

File: plugins/org.jkiss.dbeaver.ext.h2/pom.xml [hunk 2]
```diff
@@ -9,6 +9,6 @@
      <relativePath>../</relativePath>
   </parent>
   <artifactId>org.jkiss.dbeaver.ext.h2</artifactId>
-  <version>1.0.156-SNAPSHOT</version>
+  <version>1.0.157-SNAPSHOT</version>
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.ext.h2/pom.xml]: The same

**Review by serge-rider — CHANGES_REQUESTED** [review]

File: plugins/org.jkiss.dbeaver.ext.h2/META-INF/MANIFEST.MF (hunk 1 above)
Comment (EmYar) [reply plugins/org.jkiss.dbeaver.ext.h2/META-INF/MANIFEST.MF]: Change reverted

File: plugins/org.jkiss.dbeaver.ext.h2/pom.xml (hunk 2 above)
Comment (EmYar) [reply plugins/org.jkiss.dbeaver.ext.h2/pom.xml]: Change reverted

**Review by ShadelessFox — APPROVED** [review]

File: bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java [hunk 3]
```diff
@@ -1047,4 +1047,19 @@ public static String normalizeResourcePath(@NotNull String resourcePath) {
         resourcePath = resourcePath.replace('\\', '/');
         return resourcePath;
     }
+
+    /**
+     * Replaces the last {@code toReplace} entry in {@code string} with {@code replacement}
+     */
+    @NotNull
+    public static String replaceLast(@NotNull String string, @NotNull String toReplace, @NotNull String replacement) {
```
Comment (ShadelessFox) [inline bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java:1055]: Why is it here? Bad merge commit?

Comment (EmYar) [reply bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java:1055]: This is a utility method for String used by the H2 migrator. Is this not the right place for it?

Comment (ShadelessFox) [reply bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java:1055]: Yes, since it's not used anywhere in this PR.

**Review by serge-rider — APPROVED** [review]

**Pull request merged**